src/goto-symex/symex_start_thread.cpp

```diff
@@ -67,16 +67,16 @@ void goto_symext::symex_start_thread(statet &state)
     ssa_exprt lhs(pair.second.first.get_original_expr());
 
     // get L0 name for current thread
-    lhs.set_level_0(t);
-    const irep_idt &l0_name = lhs.get_identifier();
+    const renamedt<ssa_exprt, L0> l0_lhs =
+      symex_level0t{}(std::move(lhs), ns , t);
```

Comment:
So it's default constructed then `operator()`'d? Could that be improved? Perhaps comment what's going on?

Reply:
I've made this functor a function instead so it should be better